internal/mikrotik/record.go

```diff
@@ -60,22 +60,28 @@ func NewRecordFromEndpoint(endpoint *endpoint.Endpoint) (*DNSRecord, error) {
 func NewEndpointFromRecord(record DNSRecord) (*endpoint.Endpoint, error) {
 	log.Debugf("converting Mikrotik DNS record: %v", record)
 
+	recType := record.Type
+	// "" means A record because mikrotik is weird like that... :P
```

Comment:
the comment is redundant imho since it just reads precisely what the code says underneath. we can remove it since having the if statement this way is self-documenting